pytest_zigzag/data/schema/pytest-zigzag-config.schema.json

```diff
@@ -0,0 +1,48 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "pytest-zigzag-config",
+  "description": "A JSON schema for pytest-zigzag configuration files.",
+  "type": "object",
+  "required": ["environment_variables"],
+  "properties": {
+    "environment_variables": {
+      "description": "A white list of environment variables to extract and include in the JUnitXML global properties.",
+      "type": "object",
+      "properties": {
+        "BUILD_URL": {"type": ["string", "null"]},
+        "BUILD_NUMBER": {"type": ["number", "null"]},
+        "BUILD_ID": {"type": ["string", "null"] },
+        "MODULE_HIERARCHY": {"type": ["array", "null"] },
```

Comment:
Having the "MODULE_HIERARCHY" be under "environment_variables" is a little confusing given it is specific to qTest and not the environment in which the tests were ran. I suggest making a top-level "qtest" container as a peer to the "environment_variables" container.